deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir[File.join(options[:metadata_path], "app_icon.{png,jpg}")].first
```

Comment:
This line can return `nil`, which `File.exist?` on the following line will choke on with `no implicit conversion of nil into String (TypeError)`. PR incoming…